source/docs/advanced/production-ready/media-middleware.md

```diff
@@ -102,7 +102,7 @@ This component will automatically fetch the image through the proxy with the cor
 
 It will also lazyload the image. However, in some cases you might not want this to happen. For instance, you don't want this to happen on the main image of your product page. To disable lazyloading, you can use the option `dangerouslyDisableLazyLoad` or (even better in this example) **the `priority` prop that will also add a `<link rel="preload">` to the page.**
 
-For a more data-efficient browser preload, we also recommend that you define the [`sizes` HTML attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-sizes) for the image. Here is an example:
+For a more data-efficient browser preload, we also recommend that you define the [`sizes` HTML attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-sizes) for the image. A more detailed explanation about the `sizes` attribute and how to determine the best `sizes` for your image check out [this](https://ericportis.com/posts/2014/srcset-sizes/#part-2) article. Also see [below](#image-sizes) how we added `sizes` in our themes and how and when you should override it. Here is an simple example of the `sizes` attribute:
```

Comment:
What we've tried to achieve across the documentation was:
- keep sentences short
- avoid as much as possible adjectives such as "easy", "simple"… and explain things like if the person didn't know the basics (often linking to an existing page is enough)
- use meaningful text for links (instead of "this", "here"…)

I tried to suggest some minor changes to illustrate this.

```suggestion
For a more data-efficient browser preload, we also recommend that you define the [`sizes` HTML attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-sizes) for the image. A more detailed explanation about the `sizes` attribute and how to determine the best `sizes` for your image check out this [srcset + sizes = AWESOME!](https://ericportis.com/posts/2014/srcset-sizes/#part-2) article. [The "Images size" section below](#image-sizes) explains how we added `sizes` attributes in our themes, and how and when you should override it. Here is an example of the `sizes` attribute:
```